parastell/parastell.py

```diff
@@ -622,10 +660,16 @@ def parastell():
     
     if args.ivb or args.magnets:
         dagmc_export = all_data['dagmc_export']
+        stellarator.build_cubit_model(
+            **(filter_kwargs(dagmc_export, build_cubit_model_allowed_kwargs)))
         stellarator.export_dagmc(
             export_dir=args.export_dir,
-            **dagmc_export
+            **(filter_kwargs(dagmc_export, export_allowed_kwargs))
         )
+        cub5_export = all_data['cub5_export']
+        if cub5_export['export_cub5']:
```

Comment:
Why do we need a `export_cub5` flag if we're including a separate `cub5_export` dictionary in the YAML file?

Reply:
I'm intending this to be optional behavior, but since there are no other args to get passed having the dict doesn't really make sense, I'll change it to just be a single element